Sources/Sentry/SentryNetworkTracker.m

```diff
@@ -440,6 +450,11 @@ - (void)captureFailedRequests:(NSURLSessionTask *)sessionTask
     }
 
     context[@"response"] = response;
+
+    if (self.isGraphQLOperationTrackingEnabled) {
+        context[@"operation_name"] = [sessionTask getGraphQLOperationName];
```

Comment:
`m`: On Java, they have more information for the context.. Let's prefix this with for now until we can provide more information.
```suggestion
        context[@"graphql_operation_name"] = [sessionTask getGraphQLOperationName];
```